jx_base/queries.py

```diff
@@ -28,3 +28,19 @@ def is_variable_name(value):
     if not match:
         return False
     return match.group(0) == value
+
+def dequote(s):
+    """
+    If a string has single or double quotes around it, remove them.
+    Make sure the pair of quotes match.
+    If a matching pair of quotes is not found, return the string unchanged.
+    """
+    if (s[0] == s[-1]) and s.startswith(("'", '"')):
+        return s[1:-1]
+    return s
+
+def is_column_name(col):
+    if re.match(r"(\$|\w|\\\.)+(?:\.(\$|\w|\\\.)+)*\.\$\w{6}$", col):
```

Comment:
Good point here!  We have `is_variable_name` that tests for legitimate document property path names, but they fail on the SQL column names.  I am having the same issue on the Elasticsearch side.  Should these be separate methods, like you have made, or should `is_variable_name` be more flexible?  I think I prefer your solution, but it is more code.